samples/graph-activity-feed-broadcast/nodejs/views/BroadcastNotification.ejs

```diff
@@ -36,80 +36,80 @@
+        microsoftTeams.app.getContext().then((context) => {
+            userId = context.user.id;
+            userName = context.user.userPrincipalName;
 
-            if(context.subEntityId!="" && context.subEntityId!= undefined)
-            {
+            if (context.page.subPageId != "" && context.page.subPageId != undefined) {
                 let taskInfo = {
                     title: "Broadcast details",
-                    height: 400,
-                    width: 400,
-                    url: window.location.origin + "/BroadcastDetails?taskId=" + context.subEntityId,
-                    card: null,
-                    fallbackUrl:null,
-                    completionBotId: null,
-                    };
-                microsoftTeams.tasks.startTask(taskInfo);
+                    size: {
+                        height: 400,
+                        width: 400
+                    },
+                    url: window.location.origin + "/BroadcastDetails?taskId=" + context.page.subPageId
+                };
+
+                microsoftTeams.dialog.open(taskInfo);
             };
         });
 
         // Send notification to group chat for task creation.
         function SendNotificationToOrganisation() {
             var isValid = true;
-             $('#title,#description').each(function () {
-                 if ($.trim($(this).val()) == '') {
-                     isValid = false;
-                     $(this).css({
-                         "border": "1px solid red"
-                     });
-                 }
-                 else {
-                     $(this).css({
-                         "border": "",
-                         "background": ""
-                     });
-                 }
-             });
+            $('#title,#description').each(function () {
+                if ($.trim($(this).val()) == '') {
+                    isValid = false;
+                    $(this).css({
+                        "border": "1px solid red"
+                    });
+                }
+                else {
+                    $(this).css({
+                        "border": "",
+                        "background": ""
+                    });
+                }
+            });
 
-             if (isValid == false) {
-                 e.preventDefault();
-                 return false;
-             }
+            if (isValid == false) {
+                e.preventDefault();
+                return false;
+            }
 
             let taskInfo = {
-                 id: data.length + 1,
-                 title: $('#title').val(),
-                 description: $('#description').val(),
-                 userId: userId,
-                 userName: userName
-             };
+                id: data.length + 1,
+                title: $('#title').val(),
+                description: $('#description').val(),
+                userId: userId,
+                userName: userName
+            };
```

Comment:
Please add broadcast method related doc in readme file in further reading section.